components/menu/menu-item-styles.js

```diff
@@ -4,34 +4,29 @@ import { css } from 'lit';
 export const menuItemStyles = css`
 	:host {
 		background-color: var(--d2l-menu-background-color);
-		border: 1px solid transparent;
```

Comment:
Note: this change reduces the height of the menu-items by `1px`. I confirmed with Jeff that this is ok. It does result in a few extra visual-diffs.